fastlane/lib/fastlane/actions/update_fastlane.rb

```diff
@@ -31,7 +31,7 @@ def self.run(options)
           UI.important("It seems that your Gem directory is not writable by your current user.")
           UI.important("fastlane would need sudo rights to update itself, however, running 'sudo fastlane' is not recommended.")
           UI.important("If you still want to use this action, please read the documentation on how to set this up:")
-          UI.important("https://docs.fastlane.tools/actions/#update_fastlane")
+          UI.important("https://docs.fastlane.tools/actions/update_fastlane")
```

Comment:
A minor change, but can you please put the `/` at the end? 🙏 

```suggestion
          UI.important("https://docs.fastlane.tools/actions/update_fastlane/")
```

Reply:
Thank you @mollyIV 
I committed your suggestion